packages/react-components/src/components/DetailsCard/DetailsCard.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+
+import noop from '../../utils/noop';
+import { Avatar } from '../Avatar';
+import { Button } from '../Button';
+import { PromoBannerV2 } from '../PromoBannerV2';
+import { Tag } from '../Tag';
+import { Heading } from '../Typography';
+
+import image from './assets/clock-light.svg';
+import { DetailsCard } from './DetailsCard';
+import { DetailsCardInfo } from './DetailsCardInfo';
+
+import './DetailsCard.stories.css';
+
+export default {
+  title: 'Components/DetailsCard',
+  component: DetailsCard,
+};
+
+export const Default = (): React.ReactElement => {
```

Comment:
I think we can divide it into separate stories, so we can be consistent with already existing examples.